src/main/java/client/BaseHttpClient.java

```diff
@@ -0,0 +1,10 @@
+package client;
+
+public class BaseHttpClient {
+
+  private static String BASE_URL = "https://stellarburgers.nomoreparties.site/";
+
+  public static String getBASE_URL() {
```

Comment:
```suggestion
  public static String getBaseUrl() {
```
В названии методов не используются нижнее подчеркивание